go/internal/ocxcli/doctor_coordinator.go

```
@@ -0,0 +1,278 @@
+	for _, suffix := range []string{"-journal", "-wal", "-shm"} {
+		if _, err := os.Lstat(path + suffix); err == nil {
+			return DoctorCoordinatorDiagnostic{Kind: "unsafe", Path: path, Reason: "the coordinator has an active SQLite " + suffix[1:] + " sidecar"}
+		}
+	}
+	db, err := sql.Open("sqlite", "file:"+path+"?mode=ro&immutable=1")
+	if err != nil {
+		return DoctorCoordinatorDiagnostic{Kind: "unreadable", Path: path, Reason: err.Error()}
+	}
+	defer db.Close()
+	var version int
+	if err = db.QueryRow("PRAGMA user_version").Scan(&version); err != nil {
```

golangci-lint (2.13.2)

[error] (*database/sql.DB).QueryRow must not be called. use (*database/sql.DB).QueryRowContext

(noctx)